packages/cli/src/commands/import/plan.ts

```
@@ -0,0 +1,1340 @@
+          } catch (err) {
+            this.style.actionFailure()
+
+            throw err
+          }
+
+          return
+        }
+        case 'new': {
+          return
+        }
+        case 'exit':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)